src/OrbitQt/HistogramWidget.cpp

```diff
@@ -303,8 +369,6 @@ static void DrawHint(QPainter& painter, int width, orbit_display_formats::TimeUn
                     kHintSecondLineColor);
 }
 
-namespace orbit_qt {
```

Comment:
Why you are erasing this? 
And why you are not erasing the closing one?

Reply:
I have only moved it all the way up.